cmd/backup-manager/app/util/util.go

```diff
@@ -266,6 +271,74 @@ func GetOptions(provider v1alpha1.StorageProvider) []string {
 	}
 }
 
+/*
+	GetCommitTsFromMetadata get commitTs from mydumper's metadata file
+
+	metadata file format is as follows:
+
+		Started dump at: 2019-06-13 10:00:04
+		SHOW MASTER STATUS:
+			Log: tidb-binlog
+			Pos: 409054741514944513
+			GTID:
+
+		Finished dump at: 2019-06-13 10:00:04
+*/
+func GetCommitTsFromMetadata(backupPath string) (string, error) {
```

Comment:
Can you add a unit test function for this function?

Reply:
added in https://github.com/pingcap/tidb-operator/pull/2899/commits/b6e31209b85f6ed44d286276031925c41a4be132